common/scala/src/main/resources/logback.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
   <jmxConfigurator></jmxConfigurator>
+  <include optional="true" resource="whisk-logback.xml"/>
```

Comment:
I guess this is for local debugging and should not be checked in? If so, maybe add it to .gitignore and .dockerignore

Reply:
Yes this is meant to allow other services based om common module to customize the logback config